Makefile.in

```diff
@@ -94,7 +94,7 @@ declare :
 	eups declare --root $(EUPS_DIR) eups $$vers && \
 	echo eups declare --root $(EUPS_DIR) eups $$vers
 show : git.version
-	@echo "Eups will use                               @EUPS_PYTHON@"
+	@echo "Eups will use                               $(EUPS_PYTHON)"
```

Comment:
so the `.in` processing doesn't replace `@EUPS_PYTHON@`?

Reply:
It does, but it has already been interpolated above.  Its easier for manual editing/testing to have the value only hard coded in the resulting `Makefile` once.

Reply:
Ok. I agree. I see that the interpolation is done at the top of the file and it is consistent with what I see in other makefiles where they use interpolation for the setting of the variable at the top but don't use interpolation lower down.